sys/vm/vm_map.c

```diff
@@ -185,7 +185,7 @@ static void
 vm_map_log(const char *prefix, vm_map_entry_t entry)
 {
 	char buffer[128];
-	char prt[4];
+	char prt[6];
```

Comment:
I think this is no longer needed?

Reply:
Oh, it is, I just have a bug below in that I'm writing 'R' and 'W' to the wrong places.